artifacthub/library/general/allowedrepos/1.0.0/artifacthub-pkg.yml

```diff
@@ -0,0 +1,22 @@
+version: 1.0.0
+name: k8sallowedrepos
+displayName: Allowed Repositories
+createdAt: "2022-09-16T16:22:28Z"
+description: Requires container images to begin with a string from the specified list.
+digest: c8ab2f7f-56b2-4779-8e81-8badb782dca7
```

Comment:
what is this digest?

Reply:
The digest is the way AH decides whether to refresh a package or not. It will only refresh the package if the digest is updated since the last refresh.

Reply:
updated digest to CT hash. This way changes in CT will trigger refresh on AH